code/__HELPERS/string_lists.dm

```diff
@@ -0,0 +1,14 @@
+GLOBAL_LIST_EMPTY(string_lists)
+
+/**
+ * Caches lists with non-numeric stringify-able values (text or typepath).
+ */
+/proc/string_list(list/values)
```

Comment:
can you explain to me why is this needed or better?

Reply:
it automatically caches the list to a string key